lib/fast_project_serializer.rb

```diff
@@ -0,0 +1,72 @@
+class FastProjectSerializer
```

Comment:
we could probably leverage the v1 json api serialiser @marten built, e.g. https://github.com/zooniverse/Panoptes/blob/9826fe1f9f82b3ba2efac914319529a9429ee628/lib/kafka_classification_serializer.rb